app.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-plusplus */
+/* eslint-disable linebreak-style */
+// Implement the logic of the mobile menu
+// the user clicks (or taps) the hamburger button, the mobile menu appears.
```

Comment:
- [ ] Please, consider removing comments  - this is a bad practice. Let us make a code clear 👍 
- [ ] Please, also try to modify the styling of your mobile menu. Here on the screenshot, I made some comments to improve, considering the original Figma design:
> You design (with my comments)
<img width="458" alt="Screen Shot 2022-04-05 at 20 51 28" src="https://user-images.githubusercontent.com/49064106/161814415-c3f94752-f9a0-4c9e-a208-3253f6e3437f.png">

> Figma design
<img width="267" alt="Screen Shot 2022-04-05 at 20 53 46" src="https://user-images.githubusercontent.com/49064106/161814446-c0ef5541-7e53-4c9e-b0d8-46c9502130d9.png">

Reply:
**The changes are implemented, Thank you for the professional review.**